src/openMRS.yaml

```diff
@@ -0,0 +1,45 @@
+---
+description: >
+  A collection and combination of multiple different tools for all possible
+  communication means for Nonprofits.
+  These include Auto-Response, Broadcast, Conference Line, Forwarder,
+  Ringdowns, Surveys and more.
+# the license should be OSI approved:
+# https://opensource.org/licenses/alphabetical
+license: MIT
+name: Twilio.org Rapid Response Kit
+programmingLanguages:
+  - python
+  - html
+rating: 3
+repoUrl: "https://github.com/Twilio-org/rapid-response-kit"
+sdgs: [17]
+websiteUrl: "https://www.twilio.org/"
+
+contributionGuidelinesUrl: "https://github.com/Twilio-org/rapid-response-kit/\
+                            blob/master/contributors.md"
+logoUrl: "https://www.twilio.org/wp-content/themes/twilio/assets/images/\
+          icon-twilio.sv://www.twilio.org/wp-content/themes/twilio/assets/\
+          images/icon-twilio.svg"
+starsUrl: "https://img.shields.io/github/stars/Twilio-org/rapid-response-kit.\
+           svg?style=social&label=Star&maxAge=2592000"
+numberContributors:
+  url: "https://api.github.com/repos/Twilio-org/rapid-response-kit/stats/\
+        contributors"
+  format: json  # or yaml or xml
+  accessor: list  # or number or property name prepended by ?
+
+ratingCommend: Due to longer inactivity and slightly lacking contribution guide
+otherLinks:
+  - name: Awesome Twilio
+    descrption: >
```

Comment:
description*   Please be careful about the variable typos :rofl:

Reply:
Can you also update the TEMPLATE.yaml file (and rename it to PROJECT_TEMPLATE.yaml as to not confuse with README_TEMPLATE.md which I added in my PR? I can do this in my branch if you prefer, just trying to avoid a git conflict)

Reply:
Wow 2/2 ^^'

I would do that in an extra PR since I don't fell like it should be part of a project PR